x-pack/plugins/session_view/public/components/process_tree/hooks.ts

```diff
@@ -170,15 +170,16 @@ export class ProcessImpl implements Process {
   // to be used as a source for the most up to date details
   // on the processes lifecycle.
   getDetailsMemo = memoizeOne((events: ProcessEvent[]) => {
+    // TODO: add these to generator
     const actionsToFind = [EventAction.fork, EventAction.exec, EventAction.end];
     const filtered = events.filter((processEvent) => {
-      return actionsToFind.includes(processEvent.event.action);
+      return true;
```

Comment:
Why we still need that filter?

Reply:
the this.events array contains all events for a particular process, which can include many event.action. e.g fork, exec, end, session_id_change, uid_change, etc... Not only that but any alert events will be stored in this.events as well (something we plan to break out into it's own this.alerts array). I assume this change was done while testing mock data perhaps? I believe @zizhouW will be making changes here regardless soon.

Reply:
It might be worth updating process_events_route.ts to filter out the event.action types we don't need. like uid_change, gid_change, session_id_change etc... if we do that and store alerts in it's own array then we can do away with this filter.

Reply:
ya exactly this change is not intended to be kept, was just for working with mock data